### PR TITLE
input custom class fix

### PR DESCRIPTION
**Pull request by BigOHenry**

### CODE REVIEW

File: src/Inputs/UploadInput.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function getControl()
 	{
 		/** @var Html $control */
 		$control = parent::getControl();
-		$control->class = 'custom-file-input';
+		trim($control->class .= ' custom-file-input');
```
Comment (dakorpar) [inline src/Inputs/UploadInput.php:50]: trim returns string, does not have any side effects. In this case return value is not assigned to anything... This code does not make sense...

**Review by dakorpar — CHANGES_REQUESTED** [review]

File: src/Inputs/UploadInput.php (hunk 1 above)
Comment (BigOHenry) [reply src/Inputs/UploadInput.php:50]: yeah i know and i am idiot...

**Comment by BigOHenry** [comment]
If this is not still OK, please fix it... Sadly I dont have time and then i make these basic errors. Sorry

File: src/Inputs/UploadInput.php (hunk 1 above)
Comment (BigOHenry) [reply src/Inputs/UploadInput.php]: it should be ok now

**Comment by BigOHenry** [comment]
Thank you!

**Pull request merged**